packages/muelu/test/scaling/Driver.cpp

```diff
@@ -472,7 +481,7 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
         Btpetra = Teuchos::rcp(& Xpetra::toTpetra(*B),false);
       }
 #endif
-#if defined(HAVE_MUELU_EPETRA)
+#if defined(HAVE_MUELU_EPETRA) && !defined(HAVE_MUELU_INST_COMPLEX_INT_INT)
```

Comment:
Also `HAVE_MUELU_INST_COMPLEX_INT_LONG_LONG`?

Reply:
MueLu does not do the correct stuff at present if Epetra is on and GO=INT is off. cf. #3428

Reply:
Fair point